docs/en_US/CommunitySharings/HpoComparision.md

```diff
@@ -1,4 +1,4 @@
-# Hyperparameter Optimization Comparison
+# Hyper Parameter Optimization Comparison
 *Posted by Anonymous Author*
 
 Comparison of Hyperparameter Optimization algorithms on several problems.
```

Comment:
... Hyperparameter Optimization (HPO) algorithms...

Reply:
fixed and I put this pr's change together with the #1407pr  https://github.com/microsoft/nni/pull/1407